tests/test-ctlevent.js

```diff
@@ -1,17 +1,19 @@
 var assert = require('assert');
-var CTLEvent = require('../ctlevent.js');
+var CTLEvent = require('../ctlevent.js').CTLEvent;
+var fs = require('fs');
 
 describe('CTLEvent', function() {
+    var json = JSON.parse(fs.readFileSync('./tests/data.json', 'utf8'));
+    var events = json.bwEventList.events;
+
     describe('constructor', function() {
-        console.log(CTLEvent.CTLEvent);
         it('should not fail when given empty data', function() {
-            var e = CTLEvent.CTLEvent({});
+            var e = new CTLEvent({});
```

Comment:
@nbuonin I made a stupid mistake here yesterday - when creating instances in JavaScript we need to say `new CTLEvent()`, not just `CTLEvent()`, because then it's just a function.